idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -975,83 +960,43 @@ def rule_yb_CO2(blk, t, x):
-                return Expression.Skip
-            else:
-                return (
-                    1 + (blk.liquid_phase.properties[t, x].diffus['MEA'] *
-                         blk.liquid_phase.properties[
-                             t, x].conc_mol_comp_true['MEA']) *
-                    (1 - blk.yi_solvent[t, x, j] * blk.yi_solvent[t, x, j]) /
-                    (2 * blk.liquid_phase.properties[t, x].diffus['MEACOO-'] *
-                     blk.liquid_phase.properties[
-                         t, x].conc_mol_comp_true['MEACOO-']))
-
-        self.yi_MEACOO = Expression(
-            self.flowsheet().time,
-            self.liquid_phase.length_domain,
-            solvent_comp_list,
-            rule=rule_yi_MEACOO,
-            doc='Dimensionless concentration of MEACOO-')
-
-        # def rule_yi_MEAH(blk, t, x):
-        #     if x == self.liquid_phase.length_domain.last():
-        #         return Expression.Skip
-        #     else:
-        #         return (
-        #             1 + (blk.liquid_phase.properties[t, x].diffus['MEA'] *
-        #                  blk.liquid_phase.properties[
-        #                      t, x].conc_mol_comp_true['MEA']) *
-        #             (1 - blk.yi_MEA[t, x] * blk.yi_MEA[t, x]) /
-        #             (2 * blk.liquid_phase.properties[t, x].diffus['MEA+'] *
-        #              blk.liquid_phase.properties[
-        #                  t, x].conc_mol_comp_true['MEA+']))
         
-        def rule_yi_MEAH(blk, t, x, j):
-            if x == self.liquid_phase.length_domain.last():
+        def rule_yi_electrolyte_diffus(blk, t, x, j, k):
+            if x == self.liquid_phase.length_domain.last() or j == 'H2O':
                 return Expression.Skip
             else:
                 return (
-                    1 + (blk.liquid_phase.properties[t, x].diffus['MEA'] *
+                    1 + (blk.liquid_phase.properties[t, x].diffus[j] *
                          blk.liquid_phase.properties[
-                             t, x].conc_mol_comp_true['MEA']) *
+                             t, x].conc_mol_comp_true[j]) *
                     (1 - blk.yi_solvent[t, x, j] * blk.yi_solvent[t, x, j]) /
-                    (2 * blk.liquid_phase.properties[t, x].diffus['MEA+'] *
+                    (2 * blk.liquid_phase.properties[t, x].diffus[k] *
                      blk.liquid_phase.properties[
-                         t, x].conc_mol_comp_true['MEA+']))
+                         t, x].conc_mol_comp_true[k]))
 
-        self.yi_MEAH = Expression(self.flowsheet().time,
+        self.yi_electrolyte_diffus = Expression(self.flowsheet().time,
                                   self.liquid_phase.length_domain,
                                   solvent_comp_list,
-                                  rule=rule_yi_MEAH,
-                                  doc='Dimensionless concentration of MEA+')
+                                  electrolyte_comp_diffus,
```

Comment:
This is going to need some more thought and effort - for now I don't think these are actually needed as they are only used to calculate the enhancement factor.

Moving forwards, I think there should only be a single interfacial concentration (i.e. not separate ones for solvents and electrolytes), with different `if/else` branches based on component type. However, we also need to look into exactly how these concentrations are calculated - this might just need to be part of the "enhancement factor" sub-model that users might need to provide.